pkgs/top-level/python-packages.nix

```
@@ -4980,6 +4980,8 @@ in {
   python-simple-hipchat = callPackage ../development/python-modules/python-simple-hipchat {};
   python_simple_hipchat = self.python-simple-hipchat;
 
+  scs = callPackage ../development/python-modules/scs { };
```

to avoid infinite recursion
```suggestion
  scs = callPackage ../development/python-modules/scs { scs = pkgs.scs; };
```

---

True, thanks.